backtracking/subarray_sum.cpp

```diff
@@ -14,6 +14,7 @@
  */
 
 #include <cassert>        /// for assert
+#include <cstdint>        /// for integral typedefs
```

Comment:
The comment is not too useful. It could well be something like
```suggestion
#include <cstdint>        /// included because it is needed
```
I would suggest, in case of this file, to
```suggestion
#include <cstdint>        /// for int_64_t and uint64_t
```
I realize that it might be hard to do it by hand. [include-what-you-use](https://github.com/include-what-you-use/include-what-you-use) might help to automate this process.

On the other hand: I am not sure if these comments are needed at all. Having in mind that this change is very urgent (79 files do not compile), I would just skip the comments.

By the way: these types are called [_fixed width integer types_](https://en.cppreference.com/w/cpp/types/integer).

Reply:
> The comment is not too useful. It could well be something like
> ```suggestion
> #include <cstdint>        /// included because it is needed

Yes I do agree

> ```
> I would suggest, in case of this file, to
> ```suggestion
> #include <cstdint>        /// for int_64_t and uint64_t

We'd have to look through the entire file
But in general is probably just uint64_t

> ```
> I realize that it might be hard to do it by hand. [include-what-you-use](https://github.com/include-what-you-use/include-what-you-use) might help to automate this process.

I'll have a look into this. Does this provide this type of comment or
A different style?

> On the other hand: I am not sure if these comments are needed at all. Having in mind that this change is very urgent (79 files do not compile), I would just skip the comments.

Oversight on my part. Some of the files listed does not have comments on the other includes. But on the other hand, not including them would contribute to 
#2546


> By the way: these types are called [_fixed width integer types_](https://en.cppreference.com/w/cpp/types/integer).

Oh thanks for letting me know 🙂

Reply:
> I'll have a look into this. Does this provide this type of comment or A different style?

The _full_ output of for this file looks like that:
```
#include <cassert>        // for assert
#include <cstdint>        // for int64_t, uint64_t
#include <iostream>       // for operator<<, cout, endl, ostream, basic_ostream
#include <unordered_map>  // for unordered_map, operator!=, unordered_map<>::iterator, _Node_iterator_base
#include <vector>         // for vector
```

Skipping one of the `/` might be actually a good idea, cf. #2493.